notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -0,0 +1,1530 @@
+{
```

Comment:
<p>I'm not sure about <code>capturing the high-order interactions between..</code>. Even though in theory, FM can be of order &gt;3, in practice it can be time consuming. That's why the DeepFM and xDeepFM were created. The FM will address the low order interactions (1 and 2) and the DNN will address higher interactions (&gt;3).</p><p> </p><p>Maybe we should also link the library directly in the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yea this is a more precise statement. In practice, >2 order of FM is not used because of the time consumed for fitting a model.